README.md

```diff
@@ -1,68 +1,44 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Quiz web app
 
-## Available Scripts
+## Description:
+This is a quiz web app which helps users to check their knowledge of CYF study material.
 
-In the project directory, you can run:
+## Installation: 
+1. Clone files into your local machine using - `git clone https://github.com/CodeYourFuture/project-quiz.git`.
+2. Run `npm install` in your local terminal.
+3. Run `npm start` to run the app or open `http://localhost:3000` to view it in the browser.
 
-### `yarn start`
+## Table of Quizzes: 
```

Comment:
Let's remove the `Table of Quizzes` section since there are no quizzes right now